include/ur_client_library/ur/dashboard_client.h

```diff
@@ -78,6 +78,131 @@ class DashboardClient : public comm::TCPSocket
    */
   std::string sendAndReceive(const std::string& command);
 
+  /*!
+   * \brief Sends command and compare it with the expected answer
+   *
+   * \param command Command that will be sent to the server. It is important, that the command sent is finished with a
+   * '\n' (newline) so it will be processed by the server.
+   * \param expected Expected replay
+   *
+   * \return True if the reply to the command is as expected
+   */
+  bool sendRequest(const std::string& command, const std::string& expected);
+
+  /*!
+   * \brief brief Sends a command and wait until it returns the expected answer
+   *
+   * \param command Command that will be sent to the server
+   * \param expected Expected replay
+   * \param timeout Timeout time in seconds
+   *
+   *  \return True if the reply was as expected within the timeout time
+   */
+  bool waitForReply(const std::string& command, const std::string& expected, double timeout = 30.0);
+
+  /*!
+   * \brief Keep Sending the requesting Command and wait until it returns the expected answer
+   *
+   * \param requestCommand Request command that will be sent to the server
+   * \param requestExpectedResponse The expected reply to the request
+   * \param waitRequest The status request
+   * \param waitExpectedResponse The expected reply on the status
+   * \param timeout Timeout time in seconds
+   *
+   * \return True when both the requested command was receive with the expected reply as well as the resulting status
+   * also is as expected within the timeout time
+   */
+  bool retryCommand(const std::string& requestCommand, const std::string& requestExpectedResponse,
+                    const std::string& waitRequest, const std::string& waitExpectedResponse, unsigned int timeout);
+
+  /*!
+   * \brief Send Power off command
+   *
+   * \return True succeeded
+   */
+  bool commandPowerOff();
+
+  /*!
+   * \brief Send Power on command
+   *
+   * \param timeout Timeout in seconds
+   *
+   * \return True succeeded
+   */
+  bool commandPowerOn(unsigned int timeout = 1200);
+
+  /*!
+   * \brief Send Brake release command
+   *
+   * \return True succeeded
+   */
+  bool commandBreakeRelease();
```

Comment:
```suggestion
  bool commandBrakeRelease();
```
Make the function name the same as the dashboard command